docs/AppService.md

```diff
@@ -63,11 +66,11 @@ Run the Validation Test
 
 ```bash
 
-# run the tests in the container
-docker run -it --rm retaildevcrew/webvalidate --server https://${He_Name}.azurewebsites.net --files helium.json
+# run the tests in a container
+docker run -it --rm retaildevcrew/webvalidate --server $He_App_Endpoint --base-url https://raw.githubusercontent.com/retaildevcrews/helium/main/TestFiles/ --files baseline.json
 
 ```
 
 ## Observability
 
-See [App Service Observability](AppServiceObservability.md) for details
+See [App Observability](AppObservability.md) for details
```

Comment:
Might want to leave it as AppServiceObservability since the observability for the AKS deployment would be different.

Reply:
I'll do some more testing with the dashboards and verify whether the App Service dashboard can work with AKS.